dev-tools/audius-compose

```diff
@@ -385,6 +385,9 @@ def up(
 
     AAO_DIR = pathlib.Path(os.getenv("AAO_DIR", protocol_dir / "../anti-abuse-oracle"))
 
+    # Mediorum's Dockerfile.prod doesn't support multiple mediorum instances, so always use dev
+    os.environ['MEDIORUM_ENV_TYPE'] = 'dev'
```

Comment:
This only affects audius-cmd in CI. I don't want to do this, but it's the existing behavior. Without it, audius-cmd would use the prod build which only runs 1 mediorum instance instead of 3